.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
```

Comment:
- [ ] It's one of the project requirements to [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js), so the total checks should be 4. Please add the [linter.yml](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) file.